src/common/sourceMaps/sourceMapRepository.ts

```diff
@@ -77,10 +80,6 @@ class Directory {
         .filter(bn => bn !== 'node_modules' && bn !== '.')
         .sort()
         .map(async bn => {
-          if (!getCaseSensitivePaths()) {
-            bn = bn.toLowerCase();
-          }
```

Comment:
Rather than just lowercasing all paths, we now get the 'true' casing of the path, if necessary, before passing it into our internal logic. This ensures that all paths we return are exactly correct to what's on the filesystem for the case-sensitive CDP.